cluster/examples/kubernetes/ceph/object-test.yaml

```diff
@@ -22,3 +22,8 @@ spec:
     port: 80
     securePort:
     instances: 1
+  healthCheck:
+    # Change pod liveness probe, it work grw daemon
+    livenessProbe:
+      rgw:
```

Comment:
There is only a single daemon for the object CR, so no need for the daemon name:
```
healthCheck:
  livenessProbe:
    disabled: false
```

Reply:
Done.